<launch>
    <arg name="map_yaml_path" default="$(find planner)/config/map.yaml" />
    <arg name="camera_intrinsics_yaml_path" default="/data/config/calibrations/camera_intrinsic/$(env VEHICLE_NAME).yaml" />
    <arg name="tag_size" default="0.12" /> <!-- 120mm -->
    <arg name="render_overlay" default="true" />


    <node name="apriltag_detection_node" pkg="apriltag_detection" type="apriltag_detection_node.py" output="screen">
        <param name="map_yaml_path" value="$(arg map_yaml_path)" />
        <param name="camera_intrinsics_yaml_path" value="$(arg camera_intrinsics_yaml_path)" />
        <param name="tag_size" value="$(arg tag_size)" />
        <param name="render_overlay" value="$(arg render_overlay)" />
    </node>
</launch>
